spec/test-outputs/apt-integration.out.vcl

```diff
@@ -25,9 +25,9 @@ backend F_apt {
 sub vcl_recv {
 #FASTLY recv
 
-    # Prevent third parties from flushing CDN caches with FASTLYPURGE
+    # Require authentication for FASTLYPURGE requests unless from IP in ACL
     if (req.request == "FASTLYPURGE") {
```

Comment:
the `apt` configs don't have the ip whitelist check.  Is this intentional?  (If so, the comment might need tweaking)

Reply:
Whoops! classic copy + pasting error. Sorted now.

apt - is our own internal aptly server that we pull our software from for our ubuntu boxes.

Thanks for looking @sihugh :+1: